packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.spec.js

```diff
@@ -27,18 +43,11 @@ describe('Component > InteractionLayer', function () {
     expect(rect.prop('fill')).to.equal('transparent')
   })
 
-  it('should bind the onPointerDown prop', function () {
-    wrapper.simulate('pointerdown')
-    expect(onPointerDown).to.have.been.called()
-  })
-
-  it('should bind the onPointerMove prop', function () {
-    wrapper.simulate('pointermove')
-    expect(onPointerMove).to.have.been.called()
-  })
-
-  it('should bind the onPointerUp prop', function () {
-    wrapper.simulate('pointerup')
-    expect(onPointerUp).to.have.been.called()
+  it('should create a mark on pointer down', function () {
```

Comment:
This still needs tests for pointer move and pointer up. Maybe test that the mock mark is correctly updated by those events?

Reply:
Could this also test that the point tool mark's size has gone up by one?

Reply:
I think I've rewritten all of this in #1308, so that you can select marks as well as create them.

Reply:
>Could this also test that the point tool mark's size has gone up by one?

The tests are using a generic `MockMark` but could, I think, be changed to use an actual mark model. The Point model doesn't store any info about the marker's size, though.

Reply:
See #1308.